### PR TITLE
Add treemap to components

### LINKED ISSUE #770: integrate treemap from observablehq into plasmic
I would like to integrate this treemap on the zuzalu billboard page in plasmic: https://observablehq.com/d/c857fa5c110524ee

Observablehq says ofr Runtime with React:

This code uses the [Runtime API](https://observablehq.com/@observablehq/downloading-and-embedding-notebooks) to generate a React component. From the terminal, run:

> yarn add @observablehq/runtime@5
> yarn add "https://api.observablehq.com/d/c857fa5c110524ee.tgz?v=3"

Then copy and paste the following component into your application:

> import React, {useRef, useEffect} from "react";
> import {Runtime, Inspector} from "@observablehq/runtime";
> import notebook from "c857fa5c110524ee";
> 
> function Notebook() {
>   const chartRef = useRef();
> 
>   useEffect(() => {
>     const runtime = new Runtime();
>     runtime.module(notebook, name => {
>       if (name === "chart") return new Inspector(chartRef.current);
>     });
>     return () => runtime.dispose();
>   }, []);
> 
>   return (
>     <>
>       <div ref={chartRef} />
>       <p>Credit: <a href="https://observablehq.com/d/c857fa5c110524ee">Zuzualu hypercerts by category by hypercerts</a></p>
>     </>
>   );
> }
> 
> export default Notebook;

If you need to make changes, you can fork the treemap on observablehq (or I can add you as a collaborator if you have an account)

Let's integrate it and hide it for now @ryscheng. I will then update the data an publish it.

### PR DESCRIPTION
**Pull request by ravenac95**
Resolves #770

### CODE REVIEW

File: frontend/components/zuzalu-hypercert-treemap.compat.d.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Provided for the zuzalu-hypercert-treemap.
```
Comment (ravenac95) [inline frontend/components/zuzalu-hypercert-treemap.compat.d.ts:1]: Where should we put these? I tried to give it a reasonable name. observablehq provides no typing.

**Comment by ravenac95** [comment]
just fyi @ryscheng tried to see how one could make a dynamic loader... doesn't seem possible based on how they do the integration. They package notebooks as NPM modules

File: frontend/components/zuzalu-hypercert-treemap.compat.d.ts (hunk 1 above)
Comment (ryscheng) [reply frontend/components/zuzalu-hypercert-treemap.compat.d.ts:1]: Feel free to set the pattern :-)
We haven't had to do that yet

**Comment by ryscheng** [comment]
oh interesting, well it was worth a shot. Thanks for checking

**Pull request merged**